components/ratgdo/ratgdo.cpp

```diff
@@ -420,15 +412,17 @@ namespace ratgdo {
                 this->transmit_pending_ = true;
                 return false;
             }
-            delayMicroseconds(200);
+            delayMicroseconds(100);
         }
 
-        this->output_gdo_pin_->digital_write(true); // pull the line high for 1305 micros so the
-                                                    // door opener responds to the message
-        delayMicroseconds(1305);
-        this->output_gdo_pin_->digital_write(false); // bring the line low
+        // indicate the start of a frame by pulling the 12V line low for at leat 1 byte followed by
+        // one STOP bit, which indicates to the receiving end that the start of the message follows
+        // The output pin is controlling a transistor, so the logic is inverted
+        this->output_gdo_pin_->digital_write(true); // pull the line low for at least 1 byte
+        delayMicroseconds(1300);
+        this->output_gdo_pin_->digital_write(false); // line high for at least 1 bit
+        delayMicroseconds(130);
```

Comment:
I wonder if this will cause a problem on some of the older openers....  If so we can always adjust it if someone reports an problem

Reply:
Yes, exactly. As long as the openers use 9600 baud, I don't think they should have an issue.